helpers_test.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"fmt"
+	shutil "github.com/termie/go-shutil"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func commitAllChanges() {
+	err := run([]string{"git", "add", "-A"})
+	checkText(err, output)
```

Comment:
Where does output come from? Do we have to use global variables here? Can you return `output` from `run`?